src/ucs/arch/x86_64/cpu.c

```diff
@@ -588,6 +588,28 @@ static size_t ucs_cpu_memcpy_thresh(size_t user_val, size_t auto_val)
 }
 #endif
 
+static size_t ucs_cpu_nt_bt_thresh_min(size_t user_val)
+{
+    if (user_val != UCS_MEMUNITS_AUTO) {
+        return user_val;
+    }
+
+    if ((ucs_arch_get_cpu_vendor() == UCS_CPU_VENDOR_AMD)) {
```

Comment:
remove extra parenthesis here and below?

Reply:
Done.